Sources/CGFloatExtensions.swift

```diff
@@ -14,13 +14,8 @@ extension CGFloat {
     public var center: CGFloat { return (self / 2) }
 
     /// EZSwiftExtensions
-    public func toRadians() -> CGFloat {
```

Comment:
toRaidans does the exact same thing as degreesToRadians and they are both public methods. Its bad OOP design to have two public methods with identical functionality with one method calling the other(and not doing anything other than that).